my_sprites.py

```diff
@@ -880,6 +886,10 @@ def update(self):
             s = random.choice([s for s in Sound if s.name.startswith("FOOTSTEP_")])
             arcade.play_sound(s.value)
 
+        # Stop player from walking while attacking
+        if self.is_walking and self.equipped_weapon is not None and not self.equipped_weapon.is_idle:
+            self.all_keys_off()
```

Comment:
Maybe we could do this when the player attacks instead of checking this all the time here in _update()_?